src/Common/nunit/ColorConsole.cs

```diff
@@ -76,7 +76,6 @@ public static ConsoleColor GetColor(ColorStyle style)
                             return ConsoleColor.DarkGray;
                         case ColorStyle.Help:
                             return ConsoleColor.DarkGray;
-                        case ColorStyle.Default:
                         default:
```

Comment:
What's wrong with it?  Can you be more specific?

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/Common/nunit/ColorConsole.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28155333>:
>
> > @@ -76,7 +76,6 @@ public static ConsoleColor GetColor(ColorStyle style)
> >                              return ConsoleColor.DarkGray;
> >                          case ColorStyle.Help:
> >                              return ConsoleColor.DarkGray;
> > -                        case ColorStyle.Default:
> >                          default:
>
> Looks like a personal preference here but one that differs from the
> author. Not part of our standards.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28155333>.
>